visidata/sort.py

```diff
@@ -63,6 +63,8 @@ def sort(self):
     if self.rows is UNLOADED:
         return
     try:
+        # replace ambiguous colname strings with unambiguous Column objects  #2494
+        self._ordering = self.ordering
```

Comment:
Minor nit: let's move this to replace line 69 and just use self._ordering on line 72 where it's used.  Not a big deal but may as well save ourselves that line of code (since that's what line 69 is doing anyway).

Reply:
Okay, done.